### PR TITLE
Upgrade to 15.0.0 version

### PR DESCRIPTION
**Pull request by alepolidori**
nethserver/dev#5674

### CODE REVIEW

File: root/etc/e-smith/templates/etc/httpd/conf.d/zz_nextcloud.conf/10base [hunk 1]
```diff
@@ -14,7 +14,6 @@
 
     <IfModule mod_headers.c>
       Header always set Strict-Transport-Security "max-age=15552000"
-      Header always set Referrer-Policy "no-referrer"
```
Comment (gsanchietti) [inline root/etc/e-smith/templates/etc/httpd/conf.d/zz_nextcloud.conf/10base:17]: Why this line has been removed?

File: root/etc/e-smith/events/actions/nethserver-nextcloud-conf [hunk 2]
```diff
@@ -47,4 +47,5 @@ else
         exit 0
     fi
     OCC "db:add-missing-indices"
+    OCC "db:convert-filecache-bigint"
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-nextcloud-conf:50]: Why is this needed?

File: root/etc/e-smith/templates/etc/httpd/conf.d/zz_nextcloud.conf/10base (hunk 1 above)
Comment (alepolidori) [reply root/etc/e-smith/templates/etc/httpd/conf.d/zz_nextcloud.conf/10base:17]: Sorry for the lack of information !
Because from 14.0.4 version this warning has arised.
Nextcloud itself added the Referrer Policy in its `.htaccess` file, so a double insertion of the rule cause a warning message from the admin page of Nextcloud web app.

File: root/etc/e-smith/events/actions/nethserver-nextcloud-conf (hunk 2 above)
Comment (alepolidori) [reply root/etc/e-smith/events/actions/nethserver-nextcloud-conf:50]: This command is to optimize some database column.
It has been showed from the admin web page as a warning.
It says that this operation is not automatically done because it can be computationally intensive.
Executing it more times, does not cause any problems.

**Review by gsanchietti — APPROVED** [review]
